crates/frontend/src/components/condition_pills/types.rs

```diff
@@ -1,6 +1,122 @@
+use std::fmt::Display;
+
+use serde_json::{Map, Value};
+
+#[derive(Debug, Clone)]
+pub enum ConditionOperator {
+    Is,
+    In,
+    Has,
+    Between,
+    Other(String),
+}
+
+impl Display for ConditionOperator {
```

Comment:
Can we use strum here ?

Reply:
There was no way in strum to serialize this `Other(String)` into to just use the inner value, so went ahead with this approach.